site/docs/red-team/plugins/cca.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
This is a nit but can we please make the slug (filename) for this `context-compliance-attack`?  Better for SEO